pallets/gear/src/manager/task.rs

```diff
@@ -36,13 +36,35 @@ use gear_core::{
     message::{DispatchKind, ReplyMessage},
 };
 use gear_core_errors::{ErrorReplyReason, SignalCode};
+use sp_core::Get;
+use sp_runtime::Saturating;
 
 impl<T: Config> TaskHandler<T::AccountId> for ExtManager<T>
 where
     T::AccountId: Origin,
 {
     fn pause_program(&mut self, program_id: ProgramId) {
-        log::debug!("pause_program; id = {:?}", program_id);
```

Comment:
I wouldn't delete previously existing log. Just put it somewhere below

Reply:
it is redundant since there is already log in task processor